interfaces/cython/cantera/reactor.pyx

```diff
@@ -727,10 +727,10 @@ cdef class MassFlowController(FlowDevice):
 
 cdef class Valve(FlowDevice):
     r"""
-    In Cantera, a `Valve` is a flow devices with mass flow rate that is a
+    In Cantera, a `Valve` is a flow device with mass flow rate that is a
     function of the pressure drop across it. The default behavior is linear:
 
-    .. math:: \dot m = K_v (P_1 - P_2)
```

Comment:
I thought that we'd used two notations here that were quite confusing. In this line, the parentheses indicate multiplication. In the next (math) line, they indicate the argument of a function. I just wanted to be a bit more explicit, since I thought we might have had someone post some code that demonstrated a misunderstanding of how this worked (although now I can't recall where I saw that code, if it ever existed). I'm not sure why I chose `*` though...

It would probably be better to disambiguate this by including some text about what the symbols mean rather than only a multiplication symbol. I'll fix it and push another commit.